en/SUMMARY.md

```diff
@@ -367,6 +369,10 @@
         * [Realsense T265 Tracking Camera (VIO)](peripherals/camera_t265_vio.md)
     * [Video Streaming](companion_computer/video_streaming.md)
       * [Video Streaming using WFB-ng Wifi (Long range)](companion_computer/video_streaming_wfb_ng_wifi.md)
+  * [Static Pressure Buildup](advanced_config/static_pressure_buildup.md)
+  * [Serial Port Configuration](peripherals/serial_configuration.md)
+  * [PX4 Ethernet Setup](advanced_config/ethernet_setup.md)
```

Comment:
@sfuhrer I can't find ethernet and static pressure build up in your modified guide - does this location at the end of Hardware Setup and Configuration make sense to you?